### PR TITLE
Support page improvements for users

### PR DESCRIPTION
**Pull request by margarethaley**
This pr makes various improvements to the support page with the intention of making the page easier to navigate for users. Changes include adding a link to the github repo, moving the slack channel info closer to the top, implementing more hyperlinks for less clutter, adding bullet points to the list of managed services, and editing the descriptions to be more concise.

Resolves https://github.com/operate-first/support/issues/281

### CODE REVIEW

**Review by anishasthana — CHANGES_REQUESTED** [review]
The page says `Github` in a number of places and `GitHub` in others. We should aim for consistency :-)

Fwiw, `GitHub` seems to be the correct one to use.

**Comment by HumairAK** [comment]
Looks great! @margarethaley , the pr is failing lint checks, you can check which ones by clicking the `details` link, or running it locally on your cloned branch [as described here](https://github.com/operate-first/apps/blob/master/contributing.md#tests). 

Also, can you squash these commits down to 1 commit?

**Comment by margarethaley** [comment]
@HumairAK - I squashed them, but I don't know how to fix the failing link check. When I go into details it says that Trim Trailing Whitespace failed (not sure exactly what that means). I tried to run the tests locally according to those directions but am having trouble installing pre-commit.

File: README.md [hunk 1]
```diff
@@ -2,78 +2,67 @@
...
 
-This is the support repository for the Operate First Initiative.
+GitHub Support Repository: https://github.com/operate-first/support
 
-For any questions, concerns, and/or requests, please use the appropriate [Issue Template][1]. If no template fits your requirement, then please make a regular GitHub issue [here][2].
+For any questions, concerns, and/or requests, please use the appropriate [Issue Template][1] to open an issue on our GitHub. If no template fits your requirement, then please make a regular GitHub issue [here][2].
+
+## End User Support
+
+We have a [community slack channel](https://join.slack.com/t/operatefirst/shared_invite/zt-o2gn4wn8-O39g7sthTAuPCvaCNRnLww) where we post announcements, general information, and more. If you have any questions, feel free to post a message to the `#support` channel.
```
Comment (HumairAK) [inline README.md:11]: @margarethaley 

> When I go into details it says that Trim Trailing Whitespace failed (not sure exactly what that means) 

Trailing whitespaces: 
> Trailing whitespace is any spaces or tabs after the last non-whitespace character on the line until the newline.

So if you see at the end of line 11, there is actually a blank space. If you remove it, then it should be fine.

Comment (HumairAK) [reply README.md:11]: > I tried to run the tests locally according to those directions but am having trouble installing pre-commit. 

Feel free to reach out if you need help with this! Running the command will also apply some of these fixes automatically for you.

Comment (margarethaley) [reply README.md:11]: Ohh okay! I'll fix that - thanks so much @HumairAK!

**Review by anishasthana — APPROVED** [review]
/lgtm

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Pull request merged**